index.js

```diff
@@ -31,13 +35,16 @@ function verifyChangelog() {
     'Changelog was reduced to one line, this is an error.'
   );
   assert(
-    deletions < MAX_DELETIONS,
-    'Too many deletions (-' + deletions + '), this is probably an error.'
-  );
-  assert(
     !containsError,
     'Changelog contains an error message.'
   );
+  if (!force) {
```

Comment:
@freitagbr Let's just skip all verification with `--force`. Ideally we'll never need `--force`, but just in case we do it'll be more flexible this way